### PR TITLE
Fixed build errors

### PR DESCRIPTION
**Pull request by gabrielrinaldi**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -4,25 +4,34 @@ task test: ["test:compile", "test:cleanup"]
 desc "SASS test tasks"
 namespace :test do
   desc "Compile SASS files"
-  task :compile do
+  task compile: [:cleanup] do
+    content = ''
     Dir["*.scss"].each do |sass|
-      css = sass.sub('scss', 'css')
-      result = `sass #{sass} #{css}`
+      content += "@import \"#{sass}\";\n"
```
Comment (kllevin) [inline Rakefile]: The order the sass is imported matters.
1. Core
2. Layout
3. Components
4. Utilities

Comment (gabrielrinaldi) [inline Rakefile]: We can't dynamically do that. So should we create it manually in this case?

Comment (kllevin) [inline Rakefile]: Yes, the order won't change, and if it did the FED working on it would be responsible for updating the test :)

**Comment by gabrielrinaldi** [comment]
@chris-pearce I added the files manually to the `Rakefile` per @kllevin comment, so please remember to add any other file you create in the root there.

**Comment by kllevin** [comment]
FYI ^ @AlecRaeside

**Comment by chris-pearce** [comment]
Thanks guys, I'll merge then.

**Pull request merged**